account_payment_order/models/account_move.py

```diff
@@ -207,3 +216,25 @@ def create_account_payment_line(self):
                 }
             )
         return action
+
+    def action_payment_lines(self):
+        self.ensure_one()
+        pl_ids = (
+            self.env["account.payment.line"]
+            ._search([("move_line_id", "in", self.line_ids.ids)])
```

Comment:
You can put directly this domain in the action.

Reply:
done.